server/sonar-db-dao/src/test/java/org/sonar/db/purge/period/IntervalTest.java

```diff
@@ -77,7 +77,7 @@ public void shouldNotJoinMonthsOfDifferentYears() {
       );
 
     List<Interval> intervals = Interval.group(snapshots,
-      DateUtils.parseDateTime("2010-01-01T00:00:00+0100"), DateUtils.parseDateTime("2011-12-31T00:00:00+0100"), Calendar.MONTH);
+      DateUtils.parseDateTime("2010-01-01T00:00:00+01:00"), DateUtils.parseDateTime("2011-12-31T00:00:00+01:00"), Calendar.MONTH);
```

Comment:
Nice to have, I'm not sure date format is important in this test => we should find a way to not depend from date time here